integ-test/src/test/java/org/opensearch/sql/legacy/SQLIntegTestCase.java

```diff
@@ -745,7 +746,12 @@ public enum Index {
         TestsConstants.TEST_INDEX_GEOPOINT,
         "dates",
         getGeopointIndexMapping(),
-        "src/test/resources/geopoints.json");
+        "src/test/resources/geopoints.json"),
+    JSON_TEST(
+        TestsConstants.TEST_INDEX_JSON_TEST,
+        "json",
+        getJsonTestIndexMapping(),
+        "src/test/resources/json_test.json");
```

Comment:
Nit. As per previous comment.

```suggestion
    JSON(
        TestsConstants.TEST_INDEX_JSON,
        "json",
        getJsonIndexMapping(),
        "src/test/resources/json.json");
```